### ISSUE TITLE
Re start Wizard

### ISSUE THREAD

**Issue by ReynerHL** [cristijora/vue-form-wizard#101]
Is there anyway to reset the wizard to its initial state? I mean, how can we mark all those steps already visited as "not visited" again? Lets suppose we have a Wizard with 4 steps, and when we are in the second, then we would like to reset the wizard pressing an "Start over again" button. Thanks in advance.

**Comment by cristijora** [comment]
@ReynerHL There is no predefined method for this.
You could do it like this
```js
     this.$refs.wizard.maxStep = 0
      this.$refs.wizard.tabs.forEach((tab)=>{
     	 tab.checked = false
      })
      this.$refs.wizard.navigateToTab(0)
```
I will add a method so you can call it directly

**Comment by cristijora** [comment]
Released a new version [0.8.1](https://github.com/cristijora/vue-form-wizard/releases/tag/v0.8.1)
which has a `reset` method.
Calling it will reset the wizard to the initial state. Note that this method resets the steps only and is not responsible for what you have in your forms. 
`this.$refs.wizard.reset()`
https://jsfiddle.net/bt5dhqtf/1599/
